askama_parser/src/lib.rs

```diff
@@ -178,6 +178,8 @@ impl<I> ErrorContext<I> {
     }
 }
 
+pub(crate) type PResult<'a, T = &'a str> = IResult<&'a str, T, ErrorContext<'a>>;
```

Comment:
I'd like this defined before the `ErrorContext` type (since it relies on it), renamed to `ParseResult`, and inline the `nom::IResult` alias (that is, `ParseResult` should directly alias to something like `Result<(&'a str, T), nom::Err<ErrorContext<'a>>>`).

Reply:
So just to sure I understood: you want `PResult` to be renamed to `ParseResult` and un-alias `IResult` and move it above `ErrorContext` (as I'm writing it, I finally were able to understand your message haha).

Reply:
I want `pub(crate) type ParseResult<'a, T = &'a str> = Result<(&'a str, T), nom::Err<ErrorContext<'a>>;` to be defined before the `struct ErrorContext` (restating because I'm not sure I understand "un-alias `IResult`").

Reply:
Yes that's what I meant, replace the `IResult` alias by the aliased content (hence "un-alias", not sure if the word exists though ^^').